pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -101,6 +102,9 @@ func NewConvertCmd() *cobra.Command {
 	cmd.Flags().String("yurtctl-servant-image",
 		"openyurt/yurtctl-servant:latest",
 		"The yurtctl-servant image.")
+	cmd.Flags().String("kubeadm-conf-path",
+		"/etc/systemd/system/kubelet.service.d/10-kubeadm.conf",
+		"The path to the kubeadm configuration file that is used by the kubelet service to join the cluster. N.B. the path on different Linux distro may vary.")
```

Comment:
the parameter is not only used in cluster that managed by kubeadm tool, and other tool(like minikube, kind etc) can also use it to define the path of kubelet service conf on the edge node. so how about delete kubeadm configuration , like `The path to kubelet service conf that is used by kubelet component to join the cluster on the edge node...`